app/code/Magento/Captcha/Model/DefaultModel.php

```diff
@@ -145,6 +145,10 @@ public function getBlockName()
      */
     public function isRequired($login = null)
     {
+        if($this->captchaData->reCaptchaEnable()){
```

Comment:
No space before and after closing bracket
`if ($this->captchaData->reCaptchaEnable()) {`